content/v1.11/concepts/composition.md

```diff
@@ -917,7 +917,7 @@ Currently only `multiply` is supported.
 
 `string`. Transforms string values. 
 * string transform type `Format`, Currently only Go style fmt is supported. [Go style `fmt`][pkg/fmt] is supported.
-* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`.
+* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`, `ToJson`.
```

Comment:
```suggestion
* string transform type `Convert`, accepts one of `ToUpper`, `ToLower`, `ToBase64`, `FromBase64`, `ToJson`. `ToSha1`, `ToSha256`, `ToSha512`.
```

Reply:
done in: https://github.com/crossplane/docs/pull/406/commits/8ae216c435601a8d02aa723868992c91b0427efa